vendor/google-breakpad/src/client/linux/handler/minidump_descriptor.h

```diff
@@ -34,6 +34,7 @@
 #include <sys/types.h>
 
 #include <string>
+#include <cstdint>
```

Comment:
The change was already merged. Update your fork

Reply:
Fixed